src/style.css

```diff
@@ -1,3 +1,72 @@
+:root {
+  --primary-color: #00acdc;
+}
+
 body {
   margin: 0;
+  padding: 0;
+  box-sizing: border-box;
+}
+
+html {
+  font-family: "Roboto", sans-serif;
+  font-style: normal;
+  font-weight: 500;
+}
+
+.nav__list {
+  list-style: none;
+  margin: 0;
+  padding-right: 40px;
+  display: flex;
+  flex-direction: row;
+}
+
+.nav__item:first-child {
+  position: relative;
+}
+
+.nav__link {
+  color: #000;
+  text-decoration: none;
+  margin: 0 10px;
+  font-size: 12px;
+  display: inline-block;
+  line-height: 60px;
+
+}
+
+.nav__link:hover {
+  cursor: pointer;
+  color: #00acdc;
+}
+
+.nav {
+  display: flex;
+  justify-content: space-between;
+  text-transform: uppercase;
+  align-items: center;
+  flex-direction: row-reverse;
+}
+
+.moyo-logo {
+  position: fixed;
```

Comment:
the same with fixed
you should place this without changing position property
<img width="743" alt="image" src="https://github.com/mate-academy/layout_moyo-header/assets/102411130/cfb74aff-71de-40fa-94b5-2afe17e12344">